.github/workflows/publish-command.yml

```diff
@@ -128,45 +135,92 @@ jobs:
+          body: |
+            > :x: Couldn't auto-bump version for ${{github.event.inputs.connector}}
+      - name: Build ${{ github.event.inputs.connector }}
         run: |
-          source tools/lib/lib.sh
-          DOCKERFILE=airbyte-integrations/${{ github.event.inputs.connector }}/Dockerfile
-          echo "IMAGE_NAME=$(echo ${{ github.event.inputs.connector }} | cut -d"/" -f2)" >> $GITHUB_ENV
-          echo "IMAGE_VERSION=$(_get_docker_image_version ${DOCKERFILE})" >> $GITHUB_ENV
+          ./tools/integrations/manage.sh build airbyte-integrations/${{ github.event.inputs.connector }}
+        id: build
+        env:
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          # Oracle expects this variable to be set. Although usually present, this is not set by default on GitHub virtual runners.
+          TZ: UTC
+      - name: Test ${{ github.event.inputs.connector }}
+        if: github.event.inputs.run-tests == 'true'
+        run: |
+          ./tools/integrations/manage.sh test airbyte-integrations/${{ github.event.inputs.connector }}
       - name: Prepare Sentry
         if: startsWith(github.event.inputs.connector, 'connectors')
         run: |
           curl -sL https://sentry.io/get-cli/ | bash
       - name: Create Sentry Release
-        if: startsWith(github.event.inputs.connector, 'connectors')
+        if: startsWith(github.event.inputs.connector, 'connectors') && steps.bump-version.outputs.bumped == 'true'
         run: |
-          sentry-cli releases set-commits "${{ env.IMAGE_NAME }}@${{ env.IMAGE_VERSION }}" --auto --ignore-missing
+          sentry-cli releases set-commits "${{ env.CONNECTOR_IMAGE_NAME }}@${{ steps.bump-version.outputs.bumped_version }}" --auto --ignore-missing
         env:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_CONNECTOR_RELEASE_AUTH_TOKEN }}
           SENTRY_ORG: airbyte-5j
           SENTRY_PROJECT: airbyte-connectors
       - name: Publish ${{ github.event.inputs.connector }}
         run: |
           echo "$SPEC_CACHE_SERVICE_ACCOUNT_KEY" > spec_cache_key_file.json && docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD}
-          ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }} ${{ github.event.inputs.run-tests }} --publish_spec_to_cache
+          ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }} --publish_spec_to_cache
         id: publish
         env:
           DOCKER_USERNAME: airbytebot
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
           # Oracle expects this variable to be set. Although usually present, this is not set by default on Github virtual runners.
           TZ: UTC
+      - name: Process Build ${{ github.event.inputs.connector }}
+        run: |
+          export publish_spec_to_cache=true
+          ./tools/integrations/manage.sh process_build airbyte-integrations/${{ github.event.inputs.connector }}
```

Comment:
This basically handles the publish spec to cache and gradle processResources generator steps